src/Telegram.Bot/ITelegramBotClient.cs

```diff
@@ -58,7 +58,7 @@ public interface ITelegramBotClient
     /// <param name="request">API request object</param>
     /// <param name="cancellationToken"></param>
     /// <returns>Result of the API request</returns>
-    Task<TResponse> MakeRequestAsync<TResponse>(
+    Task<TResponse> MakeRequest<TResponse>(
```

Comment:
What do you think about renaming it to `SendRequest`? I've long thought about it since `MakeRequest` doesn't sound right to me.

Reply:
Interesting idea.
It's just that the v22 migration page helps remove the Async suffix via 3 search/replace regex, so it would make another 4th search/replace just for this... 🤔